helpers/batch_submit_SLURM.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+### Job Name (will be used as prefix later on!)
+#SBATCH --job-name="ICAR_tst"
+#SBATCH --nodes=1   
+#SBATCH --ntasks-per-node=32
+#SBATCH --time=00:05:00
+#SBATCH --constraint=haswell
+#SBATCH --qos=debug
+### Project code
+#SBATCH --account=m4062
+### error and output files in separate folder, name with jobid (%x) an job name (%j) 
+### N.B: create the job_output folder before submitting this job!
+#SBATCH --output=job_output/log-%x.%j.out
+#SBATCH --error=job_output/log-%x.%j.err
+
+# Make sure a python environment with xarray is available:
+module load python
+conda activate myenv
+
+# Set OpenMP variables
+export OMP_NUM_THREADS=1
+# export MP_TASK_AFFINITY=core:$OMP_NUM_THREADS
+
+# the easy way
+# icar icar_options.nml
+
+# the complex way (allows a continuous sequence of jobs)   
+PREFIX=tst  ##$SBATCH_JOB_NAME   
+
+# it is useful to keep all other filenames relative to $PREFIX
+# note that this is not required anywhere though
+OUTDIR=$PREFIX
+OPTFILE=options.nml  #${PREFIX}_options.nml  
+BATCHFILE=batch_submit_SLURM.sh #${PREFIX}_batch_submit.sh
+TEMPLATE=${PREFIX}_template.nml
+
+# the ICAR executable to use
+EXE=$HOME/bin/icar_dbs
+
+# load any environmental settings to run icar properly (system dependent):
+. /global/cfs/cdirs/m4062/env_scripts/UO-GNU-env.sh
+
+
+# various useful helper scripts (SETUP_RUN is critical)
+SETUP_RUN=/global/homes/b/bkruyt/icar/helpers/setup_next_run.py
```

Comment:
as long as we are modifying these files, would it make sense to make these a little more generic? i.e. replace /global/homes/b/bkruyt with ${HOME} (which works on just about any system)
MKOUTDIR below could default to `mkdir` with a commented line setting it to `mkscratch.py` (with the path expected to be set by the user)